### PR TITLE
Add option to return `idsOnly` document.

### PR DESCRIPTION
**Pull request by Czechh**
- Include full document by default
- Adds `idsOnly` option to fetch only the `ids`

### CODE REVIEW

File: src/metal.ts [hunk 1]
```diff
@@ -70,7 +70,13 @@ class MetalSDK {
       body.text = text;
     }
 
-    const { data } = await axios.post(`${API_URL}/v1/search`, body, {
+    let url = `${API_URL}/v1/search`
+
+    if (includeFullDocument) {
+      url += '?includeDoc=true'
+    }
```
Comment (softboyjimbo) [inline src/metal.ts:77]: maybe we should do this by default? wdyt?

File: tests/metal.test.ts [hunk 2]
```diff
@@ -217,8 +236,9 @@ describe('MetalSDK', () => {
       const result = await metal.tune({ idA, idB, label });
 
       expect(axios.post).toHaveBeenCalledWith(
-        `https://api.getmetal.io/v1/apps/${appId}/tunings`,
+        `https://api.getmetal.io/v1/tune`,
```
Comment (softboyjimbo) [inline tests/metal.test.ts:239]: oh damn, why was that passing?

Comment (Czechh) [reply tests/metal.test.ts:239]: it wasn't

Comment (softboyjimbo) [reply tests/metal.test.ts:239]: ...oh

**Review by softboyjimbo — APPROVED** [review]

**Pull request merged**